packages/math-input/.eslintrc.js

```diff
@@ -8,10 +8,6 @@ module.exports = {
             "error",
             {
                 packageDir: [__dirname, path.join(__dirname, "../../")],
-                // note(matthewc): I changed this in a time pinch and I'm not sure
-                // it's right so I made a ticket to circle back to it
-                // https://khanacademy.atlassian.net/browse/LC-864
-                devDependencies: ["**/*.stories.tsx", "**/*.test.tsx"],
```

Comment:
@handeyeco Weird. I can remove this in this PR and all linting passes. 🤷‍♂️

Reply:
🤷